DOCS/man/input.rst

```diff
@@ -732,6 +732,23 @@ Input Commands that are Possibly Subject to Change
 ``load-script "<path>"``
     Load a script, similar to the ``--script`` option.
 
+``change-list "<option>" "<operation>" "<value>"``
+    This command changes list options as described in `List Options`_. The
+    ``<option>`` parameter is the normal option name, while ``<option>`` is
+    the suffix or action used on the option.
+
+    Some operations take no value, but the command still requires the value
+    parameter. Int hese cases, the value must be an empty string.
+
+    .. admonition:: Example
+
+        ``change-list glsl-shaders append file.glsl``
+
+        Add a filename to the ``glsl-shaders`` list. The command line
+        equivalent is ``--glsl-shaders--apend=file.glsl`` or alternatively
```

Comment:
`--glsl-shaders-append=file.glsl`

Since `--glsl-shader` is supposed to be deprecated, I don't think it needs to be mentioned as an alternative.

Reply:
I don't think it's deprecated.

Reply:
Seems I was misremembering warnings or names on the console. I can't get them to show up for `--glsl-shader` at least.

The typo is still there though.